docs/deploying/ssl.html

```diff
@@ -0,0 +1,114 @@
+           Certificates are usually digitally signed by a certificate authority (CA) who verifies that the identity information in the certificate is correct.
+           This creates a chain of certificates between the site owner certificate and a CA certificate and transitive trust.
+            Assuming that we trust the CA, we can trust the validity of the server certificate.</p>
+       <div class="info">
+          <p class="block-text">If you want to create a CA-signed certificate chain, a useful tool to use is <a href="https://letsencrypt.org/" target="_blank">Let’s Encrypt</a>,
+           a free, automated and open CA, provided by the Internet Security Research Group (ISRG).</p>
+       </div>
+      <p>For this guide we will use OpenSSL to generate a self-signed certificate.</p>
+      <p>Create a Credentials folder in the root of your project:</p>
+      <pre><code>mkdir Credentials && cd Credentials</code></pre>
+      <p>Inside this folder we create a 2048 bit RSA private key using OpenSSL:</p>
+      <pre><code>openssl genrsa -out key.pem 2048</code></pre>
+      <p>We use this private key to create a certificate:</p>
+      <pre><code>openssl req -new -sha256 -key key.pem -out csr.csr</code></pre>
+      <p class="block-text">At this stage you provide additional information that will be stored in the certificate.</p>
+      <p class="block-text">These choices are optional except for Common Name which must be the servers hostname. In our case this is "localhost".</p>
+      <p>Next we use OpenSSL to convert the certificate to PEM format:</p>
+      <pre><code>openssl req -x509 -sha256 -days 365 -key key.pem -in csr.csr -out cert.pem</code></pre>
+      <p>Finally we encrypt the key and certificate together using a password. For simplicity we will use "password".</p>
+      <pre><code>openssl pkcs12 -export -out cert.pfx -inkey key.pem -in cert.pem</code></pre>
+<div class="underline"></div>
+      <h2 class="heading-2"><span class="blue-text">Step 2:</span> Setting up your SSL configuration</h2>
+      <p>Open your `Sources` > `Application` > `Application.swift`</p>
+      <p>We are going to use <a href="https://github.com/IBM-Swift/FileKit" target="_blank">FileKit</a> to access our certificate file.</p>
+      <p>This means we need to add FileKit to our import statements:</p>
+      <pre><code class="language-swift">import FileKit</code></pre>
+      <p>Now we initialize an `SSLConfig` struct. On macOS we use Apple Secure Transport under the covers which requires the path to our `cert.pfx` file and
+           the password we used to encrypt the file. On linux we use OpenSSL under the covers which requires the paths to the certificate and private key PEM files.</p>
+      <p>We use `#if` to check the system version and provide the appropriate requirements to our config. </p>
```

Comment:
to check the system version? don't we mean "to check which operating system we're running on"?